docs/modules/ROOT/pages/configure-clc.adoc

```diff
@@ -0,0 +1,188 @@
+
+[tabs] 
+==== 
+Platform:: 
++ 
+--
+include::partial$global-parameters.adoc[tag=!*]
+--
+Cloud:: 
++ 
+--
+include::partial$global-parameters.adoc[]
+--
+====
+
+[[tls]]
+== TLS Authentication
+
+If your cluster members use TLS to communicate with each other or clients, you need to configure Hazelcast CLC with TLS settings. Hazelcast CLC supports the following TLS authentication:
+
+- One-way authentication
+- Mutual authentication
+- No TLS verification for self-signed certificates
+
+=== Connecting to Clusters with One-Way Authentication
+
+Hazelcast Platform clusters can have one-way TLS authentication enabled. However, Hazelcast Cloud cluster always use <<mutual, mutual authentication>>.
+
+[tabs] 
+==== 
+Platform:: 
++ 
+--
+[source,yaml]
+----
+ssl:
+  enabled: true
+hazelcast:
+  cluster:
+    security:
+      credentials:
+        username: "<OPTIONAL USERNAME>"
+        password: "<OPTIONAL PASSWORD>"
+    name: "<CLUSTER-NAME>"
+    network:
+    addresses:
+    - "<MEMBER IP ADDRESS>"
+----
+--
+Cloud:: 
++ 
+--
+Not supported. See <<mutual, mutual authentication>>.
+--
+====
+
+[[mutual]]
+=== Connecting to Clusters Using Mutual TLS Authentication
+
+If clusters accept TLS connections only from trusted clients, you must provide a trusted TLS certificate for the Hazelcast CLC.
+
+[tabs] 
+==== 
+Platform:: 
++ 
+--
+[source,yaml]
+----
+ssl:
+  enabled: true
+  servername: "<HOSTNAME-FOR-CLUSTER CERTIFICATE>"
+  capath: "/tmp/ca.pem"
+  certpath: "/tmp/cert.pem"
+  keypath: "/tmp/key.pem"
+  keypassword: "<PASSWORD FOR THE KEY>"
+hazelcast:
+  cluster:
+    security:
+      credentials:
+        username: "<OPTIONAL USERNAME>"
+        password: "<OPTIONAL PASSWORD>"
+    name: "<CLUSTER-NAME>"
+    network:
+      addresses:
+        - "<MEMBER IP ADDRESS>"
+----
+--
+
+Cloud:: 
++ 
+--
+[source,yaml]
+----
+ssl:
+  enabled: true
+  capath: "/tmp/ca.pem"
+  capath: "/tmp/ca.pem"
+  certpath: "/tmp/cert.pem"
+  keypath: "/tmp/key.pem"
+  keypassword: "<PASSWORD FOR THE KEY>"
+hazelcast:
+  cluster:
+    name: "<CLUSTER NAME>"
+    cloud:
+      token: "<DISCOVERY TOKEN>"
+      enabled: true
+----
+--
+====
+
+=== Connecting to Clusters with Self-Signed TLS Certificates
+
+If your cluster members use self-signed TLS certificates, you can use the `insecureskipverify` setting to skip TLS verification and avoid connection errors.
+
+WARNING: Skipping TLS verification makes you vulnerable to man-in-the-middle attacks because any device can act as the intended cluster.
+
+[tabs] 
+==== 
+Platform:: 
++ 
+--
+[source,yaml]
+----
+ssl:
+  enabled: true
+  insecureskipverify: true
+hazelcast:
+  cluster:
+    security:
+      credentials:
+        username: "<OPTIONAL USERNAME>"
+        password: "<OPTIONAL PASSWORD>"
+    name: "<CLUSTER-NAME>"
+    network:
+    addresses:
+    - "<MEMBER IP ADDRESS>"
+----
+--
+
+Cloud:: 
```

Comment:
I think we don't need this for cloud 🤔

Reply:
Only Cloud requires the discovery token. For Platform, this config isn't needed.

Reply:
Sorry, my comment wasn't clear. I think it is better to leave out the cloud part for 'Connecting to Clusters with Self-Signed TLS Certificates'.

At least I do not know a use case for this. Of course, It may be just I'm not aware of one 😅 .